MAUI/MauiApps.sln

```diff
@@ -0,0 +1,43 @@
+﻿

```

Comment:
I would rename the folder to .NET MAUI maybe to clarify, and since it's official name, I think.

Reply:
It makes sense to keep branding. But having long folder names can cause compiler errors. I am in two minds. May be just MAUI is sufficient. @jmprieur what is your thought on branding?